<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ApplicationCentrale</class>
 <widget class="QMainWindow" name="ApplicationCentrale">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>904</width>
    <height>500</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>ApplicationCentrale</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tabWidgetCentrale">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>30</y>
      <width>901</width>
      <height>381</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabSearchCustomer">
     <attribute name="title">
      <string>Search for Customer</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tabPlanning">
     <attribute name="title">
      <string>Planning</string>
     </attribute>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>904</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File </string>
    </property>
    <widget class="QMenu" name="menuCreate_Add">
     <property name="title">
      <string>Create / Add</string>
     </property>
     <addaction name="separator"/>
     <addaction name="actionCustomer"/>
     <addaction name="actionStaff"/>
    </widget>
    <addaction name="actionExit"/>
    <addaction name="separator"/>
    <addaction name="menuCreate_Add"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>?</string>
    </property>
    <addaction name="separator"/>
    <addaction name="actionAbout"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionCustomer"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="separator"/>
   <addaction name="actionStaff"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionExit">
   <property name="icon">
    <iconset>
     <normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/exit.png</normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Exit the Application&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
  <action name="actionCustomer">
   <property name="icon">
    <iconset>
     <normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/addCustumer.png</normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/addCustumer.png</iconset>
   </property>
   <property name="text">
    <string>Customer</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Customer&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+C</string>
   </property>
  </action>
  <action name="actionStaff">
   <property name="icon">
    <iconset>
     <normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/addEmployer.jpg</normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/addEmployer.jpg</iconset>
   </property>
   <property name="text">
    <string>Staff</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Add Staff&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset>
     <normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/about.png</normaloff>../build-qt_Project_Customer-Desktop_Qt_5_10_1_MinGW_32bit-Debug/debug/Images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Information about this project and the creator&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
